tests/test_sqs/test_sqs.py

```diff
@@ -2283,3 +2283,88 @@ def test_send_message_fails_when_message_size_greater_than_max_message_size():
     ex.response["Error"]["Message"].should.contain(
         "{} bytes".format(message_size_limit)
     )
+
+
+@mock_sqs
+def test_fifo_queue_send_multiple_duplicate_messages():
+
+    sqs = boto3.resource("sqs", region_name="us-east-1")
+    msg_queue = sqs.create_queue(
+        QueueName="test-queue-dlq.fifo",
+        Attributes={"FifoQueue": "true", "ContentBasedDeduplication": "true"},
+    )
+
+    msg_queue.send_message(MessageBody="first", MessageGroupId="1")
+    msg_queue.send_message(MessageBody="first", MessageGroupId="2")
+    messages = msg_queue.receive_messages(MaxNumberOfMessages=2)
+    messages.should.have.length_of(1)
+
+
+@mock_sqs
+def test_fifo_queue_send_multiple_duplicate_messages_after_five_minutes():
+
+    sqs = boto3.resource("sqs", region_name="us-east-1")
+    msg_queue = sqs.create_queue(
+        QueueName="test-queue-dlq.fifo",
+        Attributes={"FifoQueue": "true", "ContentBasedDeduplication": "true"},
+    )
+
+    msg_queue.send_message(MessageBody="first", MessageGroupId="1")
+    time.sleep(301)
```

Comment:
Do you really want to pause the test for 5 minutes?

Would it make more sense to have the deduplication time in `models.py` as a constant instead?

Then for testing purposes, you can mock the constant to something like 10 instead?

Reply:
thanks for pointing out the approach. Did the changes.